test/potluck_test.rb

```diff
@@ -0,0 +1,137 @@
+    candy_salad = Dish.new("Candy Salad", :dessert)
+    potluck.add_dish(couscous_salad)
+    potluck.add_dish(summer_pizza)
+    potluck.add_dish(roast_pork)
+    potluck.add_dish(cocktail_meatballs)
+    potluck.add_dish(candy_salad)
+
+    assert_equal couscous_salad, potluck.get_all_from_category(:appetizer).first
+    assert_equal roast_pork, potluck.get_all_from_category(:entre).first
+    assert_equal candy_salad, potluck.get_all_from_category(:dessert).first
+  end
+
+  def test_it_returns_name_of_first_item_in_category
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    summer_pizza = Dish.new("Summer Pizza", :appetizer)
+    roast_pork = Dish.new("Roast Pork", :entre)
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+    candy_salad = Dish.new("Candy Salad", :dessert)
+    potluck.add_dish(couscous_salad)
+    potluck.add_dish(summer_pizza)
+    potluck.add_dish(roast_pork)
+    potluck.add_dish(cocktail_meatballs)
+    potluck.add_dish(candy_salad)
+
+    assert_equal "Couscous Salad", potluck.get_all_from_category(:appetizer).first.name
+    assert_equal "Roast Pork", potluck.get_all_from_category(:entre).first.name
+    assert_equal "Candy Salad", potluck.get_all_from_category(:dessert).first.name
+  end
+
+  # def test_it_has_a_menu
+  #   potluck = Potluck.new("7-13-18")
+  #   assert_equal [], potluck.menu.keys
+  # end
+
+  def test_it_sorts_dishes
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    summer_pizza = Dish.new("Summer Pizza", :appetizer)
+    roast_pork = Dish.new("Roast Pork", :entre)
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+    candy_salad = Dish.new("Candy Salad", :dessert)
+    bean_dip = Dish.new("Bean Dip", :appetizer)
+    potluck.add_dish(couscous_salad)
+    potluck.add_dish(summer_pizza)
+    potluck.add_dish(roast_pork)
+    potluck.add_dish(cocktail_meatballs)
+    potluck.add_dish(candy_salad)
+    potluck.add_dish(bean_dip)
+    menu = {:appetizers=>["Bean Dip", "Couscous Salad", "Summer Pizza"],:entres=>["Cocktail Meatballs", "Roast Pork"],:desserts=>["Candy Salad"]}
+    assert_equal menu, potluck.menu
+  end
+
+    def test_it_creates_ratio
+      potluck = Potluck.new("7-13-18")
+      couscous_salad = Dish.new("Couscous Salad", :appetizer)
+      summer_pizza = Dish.new("Summer Pizza", :appetizer)
+      roast_pork = Dish.new("Roast Pork", :entre)
+      cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+      candy_salad = Dish.new("Candy Salad", :dessert)
+      bean_dip = Dish.new("Bean Dip", :appetizer)
+      potluck.add_dish(couscous_salad)
+      potluck.add_dish(summer_pizza)
+      potluck.add_dish(roast_pork)
+      potluck.add_dish(cocktail_meatballs)
+      potluck.add_dish(candy_salad)
+      potluck.add_dish(bean_dip)
+      assert_equal 50.0, potluck.ratio(:appetizer)
+      assert_equal (1/6), potluck.ratio(:dessert)
```

Comment:
I like that you are testing the other ratios, but these assertions are expecting 0